common/OpTestUtil.py

```diff
@@ -2571,6 +2571,12 @@ def login(self, username=None, password=None):
                 self.xAuthHeader['X-Auth-Token'] = match.group(1)
                 self.jsonHeader.update(self.xAuthHeader)
             json_data = json.loads(r.text)
+            if not (json_data.get('token') is None):
```

Comment:
`if not (json_data.get('token') is None):` this same as `if json_data.get('token')` ?

Reply:
if value of (json_data.get('token') is not None --> not(True) False 
if value of (json_data.get('token') is None --> not(None) is True 

can you directly use `if json_data.get('token')`

Reply:
It won't harm, but updated, please have look